`make quick-release` fails with `KUBERNETES_CONTRIB=mesos`

I get the following error:
```
# k8s.io/kubernetes/contrib/mesos/cmd/km
contrib/mesos/cmd/km/km.go:31: undefined: NewControllerManager
Makefile:79: recipe for target 'all' failed
make[1]: *** [all] Error 1
Makefile:247: recipe for target 'cross' failed
make: *** [cross] Error 1
make: *** [quick-release] Error 1
```

This does not occur with `make` and `KUBERNETES_CONTRIB=mesos`. 

I think this was probably what initially produced #29865.

/cc @k82cn

@mtaufen , what's your build environment? Here's my output in Ubuntu 14.04:

```
...
Step 10 : ENV KUBE_OUTPUT_SUBPATH _output/dockerized
 ---> Using cache
 ---> c0882a194a46
Step 11 : ADD kube-source.tar.gz /go/src/k8s.io/kubernetes/
Untar re-exec error: exit status 1: output: link /contrib/mesos/target.sh /contrib/mesos/target.sh: no such file or directory

To retry manually, run:

docker build -t kube-build:build-2db5c8d4eb --pull=false /root/dma/kubernetes/_output/images/kube-build:build-2db5c8d4eb

Makefile:239: recipe for target 'quick-release' failed
make: *** [quick-release] Error 1
```

--- a/build/common.sh
+++ b/build/common.sh
@@ -485,11 +485,6 @@ function kube::build::source_targets() {
           \( -path ./_\* -o -path ./.git\* \) -prune  \
         \))
   )
-  if [ -n "${KUBERNETES_CONTRIB:-}" ]; then
-    for contrib in "${KUBERNETES_CONTRIB}"; do
-      targets+=($(eval "kube::contrib::${contrib}::source_targets"))
-    done
-  fi
   echo "${targets[@]}"
 }
 

